app/templates/components/public/event-map.hbs

```diff
@@ -1,8 +1,12 @@
 {{#if event.isMapShown}}
   <div class="eight wide column event-map">
     <h1>{{t 'Getting Here'}}</h1>
-    {{#g-map lat=event.latitude lng=event.longitude zoom=15 gestureHandling='cooperative' streetView='StreetViewPanorama' as |context|}}
-      {{g-map-marker context lat=event.latitude lng=event.longitude}}
+    {{#g-map markersFitMode='live' lat=37.744 lng=-122.4367 address=event.locationName zoom=2 class='google-maps' as |context|}}
```

Comment:
Why do we have hardcoded lat/lng here ?

Reply:
Removing the hardcoded lat/lng gives no map. According to the docs, there seems to be an initial value given to the component everywhere. 
https://www.emberobserver.com/addons/ember-g-map